services/brig/Dockerfile

```diff
@@ -17,8 +17,8 @@ RUN cd /src/wire-server/services/brig && make ${target}
 FROM ${deps}
 
 ARG executable
-COPY --from=builder /src/wire-server/services/brig/dist/${executable} /usr/bin/${exectuable}
-COPY --from=builder /src/wire-server/services/brig/deb/opt/brig/templates/ /usr/share/wire/templates/
+COPY --from=builder /src/wire-server/dist/${executable} /usr/bin/${exectuable}
+COPY --from=builder /src/wire-server/brig/deb/opt/brig/templates/ /usr/share/wire/templates/
```

Comment:
I think this 2nd path should remain `/src/wire-server/services/brig/deb/opt/brig/templates/`, no?